Delta/Experimental Features/Features/ShowStatusBarOptions.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  ShowStatusBarOptions.swift
+//  Delta
+//
+//  Created by Chris Rittenhouse on 4/23/23.
+//  Copyright © 2023 Riley Testut. All rights reserved.
+//
+
+import SwiftUI
+
+import DeltaFeatures
+
+struct ShowStatusBarOptions
+{
+    @Option()
```

Comment:
```suggestion
    @Option
```

Minor nit: closing parenthesis not required. (This change isn't necessary for PR to be accepted)